src/index.ts

```diff
@@ -837,6 +837,9 @@ export const plugin: PluginFunction<TypescriptMocksPluginConfig> = (schema, docu
 
     const { includedTypes, excludedTypes } = config;
     const shouldGenerateMockForType = (typeName: string) => {
+        if (!typeName) {
+            return true;
+        }
```

Comment:
I don't understand why you needed to add this check as `typeName` cannot be undefined

Reply:
Oops my bad. Im that case, can you make the function definition is updated as well (adding `typeName?`)
```typescript
const shouldGenerateMockForType = (typeName?: string) => {
```

Reply:
done